src/Psalm/Configuration.php

```diff
@@ -39,7 +39,7 @@ public static function forStrictlyCheckedNamespacesAndProjectFiles(
     }
 
     /** {@inheritDoc} */
-    public function getReportingLevelForIssue(CodeIssue $e, array $suppressed_issues = []) : string
+    public function getReportingLevelForIssue(CodeIssue $e) : string
```

Comment:
@muglug this broke BC for downstream: I know you aren't considering it that vital for now, but we'd need more stability upstream for tooling to consume psalm downstream. Maybe I could add `roave/backwards-compatibility-check` upstream, to prevent the most basic issues?

Reply:
sorry - you could add it to https://github.com/vimeo/psalm/blob/master/bin/test-with-real-projects.sh for safety

Reply:
Giving it a shot at https://github.com/vimeo/psalm/pull/2140

Progress will likely leak into next week though.